telegram/_chat.py

```diff
@@ -1894,6 +2024,78 @@ async def copy_message(
             protect_content=protect_content,
         )
 
+    async def forward_from(
+        self,
+        from_chat_id: Union[str, int],
+        message_id: int,
+        disable_notification: DVInput[bool] = DEFAULT_NONE,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = DEFAULT_NONE,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: JSONDict = None,
+    ) -> "MessageId":
+        """Shortcut for::
+
+             await bot.forward_message(chat_id=update.effective_chat.id, *args, **kwargs)
+
+        For the documentation of the arguments, please see :meth:`telegram.Bot.forward_message`.
+
```

Comment:
please add `.. seealso:: :meth:`forward_to`` before the `versionadded`. similar for `forward_to`

Reply:
I am working on it (#3110) and will add it in next pr, no?

Reply:
since this is a new method, please add it here directly - spares us some hassle of thinking about what was merged first ;)